mapper/mapper.go

```diff
@@ -470,3 +472,101 @@ func MapCookiePreferences(req *http.Request, preferencesIsSet *bool, policy *mod
 		Usage:     preferencesCookie.Policy.Usage,
 	}
 }
+
+func CreateDatasetPage(ctx context.Context, req *http.Request, d zebedee.Dataset, dlp zebedee.DatasetLandingPage, bc []zebedee.Breadcrumb, versions []zebedee.Dataset, lang string, apiRouterVersion string) datasetPage.Page {
+	dp := datasetPage.Page{}
+
+	MapCookiePreferences(req, &dp.Page.CookiesPreferencesSet, &dp.Page.CookiesPolicy)
+	dp.Type = "dataset_page"
+	dp.Metadata.Title = dlp.Description.Title
+	dp.Language = lang
+	dp.URI = d.URI
+	dp.DatasetPage.URI = dlp.URI
+	dp.Metadata.Description = dlp.Description.Summary
+	dp.DatasetPage.ReleaseDate = dlp.Description.ReleaseDate
+	dp.BetaBannerEnabled = false
+	dp.HasJSONLD = true
```

Comment:
Is it correct to have these values hard coded?

Reply:
Are these hard coded? Isn't dlp a representation of the JSON file?

Reply:
I think Carl meant the `dp.BetaBannerEnabled = false	 dp.HasJSONLD = true` values. Think we copied those over from the other page mapper originally, not sure if they're required by the parent templates, so will have to check and remove if not required....